src/index.ts

```diff
@@ -102,6 +103,59 @@ export const run = async (): Promise<void> => {
     );
   }
 
+  process.stdout.write(
+    '\nCreating test application for developing plugin...\n',
+  );
+
+  try {
+    await runSubprocess(
+      'npm',
+      [
+        'init',
+        '@capacitor/app',
+        'example',
+        '--',
+        '--name',
+        'example',
+        '--app-id',
+        'com.example.plugin',
+      ],
+      opts,
+    );
+    await runSubprocess('rm', ['-rf', './example/.git'], opts);
```

Comment:
this will fail on windows, it should use a cross-platform command or use node's fs API to delete the folder